templates/zerver/help/invite-new-users.md

```diff
@@ -1,162 +1,90 @@
 # Invite new users
 
-There are a number of ways to grant access to your Zulip organization;
-the article below describes each in more detail.
+When you invite users, you can:
 
-* Allow **anyone to join** without an invitation.
+* Set the [role](/help/roles-and-permissions) that they will have when
+  they join.
 
-* Allow people to join based on the **domain** of their email address.
+* Configure the streams they will automatically be added to.
 
-* Send **email invitations**.
+Your organization may also want to configure default
+[streams](/help/set-default-streams-for-new-users) and
+[settings](/help/configure-default-new-user-settings) for new users.
```

Comment:
We should possibly briefly express how default streams relate to invitations. Default streams are used (1) to initialize the  options offered for default streams and (2) for users who join without an invitation.

Reply:
That sounds like a good clarification to make as a follow-up on https://zulip.com/help/set-default-streams-for-new-users. I don't think we need to get into it here.

Reply:
I think I have a better proposal that I'll try drafting.

Reply:
Here's my proposal:

![image](https://user-images.githubusercontent.com/2746074/160501177-15762b39-0e02-4b2e-a018-3314d2b782a3.png)

```diff --git a/templates/zerver/help/invite-new-users.md b/templates/zerver/help/invite-new-users.md
index 2ce1430107..eb9fddbfdb 100644
--- a/templates/zerver/help/invite-new-users.md
+++ b/templates/zerver/help/invite-new-users.md
@@ -5,18 +5,20 @@ When you invite users, you can:
 * Set the [role](/help/roles-and-permissions) that they will have when
   they join.
 
-* Configure the streams they will automatically be added to.
+* Configure the initial stream subscriptions for the user. The default
+  is the organization's [default
+  streams](/help/set-default-streams-for-new-users).
 
-Your organization may also want to configure default
-[streams](/help/set-default-streams-for-new-users) and
-[settings](/help/configure-default-new-user-settings) for new users.
+Your organization may also want to configure [default
+settings](/help/configure-default-new-user-settings) for new users.
 
 
 ## Send e-mail invitations
```

Reply:
Mm, I dunno, I think it's hard to parse. Maybe keep the intro as it was, but add a sentence to the instructions step?

> Configure which streams they will be added to. The organization's default streams will be preselected.

Reply:
(Whatever we do here, the follow-up on https://zulip.com/help/set-default-streams-for-new-users needs to happen regardless.)

Reply:
Applied that edit, and agreed regarding the follow-up. @laurynmm FYI.